webservices/args.py

```diff
@@ -280,11 +280,14 @@ def make_seek_args(field=fields.Int, description=None):
 }
 
 reporting_dates = {
-    'due_date': fields.List(fields.Date, description='Date the filing is done.'),
+    'min_due_date': fields.Date(description='Date the report is due.'),
+    'max_due_date': fields.Date(description='Date the report is due.'),
     'report_year': fields.List(fields.Int, description='Year of report.'),
     'report_type': fields.List(fields.Str, description='Type of report.'),
-    'create_date': fields.List(fields.Date, description='Date this record was added to the system.'),
-    'update_date': fields.List(fields.Date, description='Date this record was last updated.'),
+    'min_create_date': fields.Date(description='Date this record was added to the system.'),
```

Comment:
If these fields are repeated exactly in other groups of args, we can break them into their own dictionary--then both views that use them can decorate their `get` methods with the added dictionary as well.